client/components/CreateNewChat/CreateNewChat.tsx

```diff
@@ -39,7 +39,6 @@ export default function CreateNewChat() {
       const data = await res.json()
 
       if (res.ok) {
-        // redirect to chatrooms screen after login
         router.push('/chatrooms')
```

Comment:
Should Line 42 in CreateNewChat.tsx be routing to '/chat-room' instead of '/chatrooms' ?